taskt/Core/Automation/Commands/Window Commands/SetWindowStateCommand.cs

```diff
@@ -54,19 +54,19 @@ public override void RunCommand(object sender)
             //loop each window and set the window state
             foreach (var targetedWindow in targetWindows)
             {
-                User32Functions.WindowState WINDOW_STATE = User32Functions.WindowState.SW_SHOWNORMAL;
+                User32Functions.WindowState WINDOW_STATE = User32Functions.WindowState.SwShowNormal;
```

Comment:
Can you rename these variables to match the coding conventions for local variables?

Reply:
As you can see that this variable is declared in a **Command File**. I was basically cleaning up the User32 Directory and so updated all the dependencies in relevant files. This will be changed when anyone of us will start cleaning these commands up.